src/mame/video/cave.cpp

```diff
@@ -1632,6 +1565,7 @@ uint32_t cave_state::screen_update_cave(screen_device &screen, bitmap_rgb32 &bit
 
 void cave_state::cave_get_sprite_info(int chip)
 {
+	int spriteram_bankmax = m_spriteram.bytes() / 0x4000;
```

Comment:
Compiling src/mame/video/cave.cpp...
../../../../../src/mame/video/cave.cpp: In member function 'void cave_state::cav
e_get_sprite_info(int)':
../../../../../src/mame/video/cave.cpp:1568:38: error: 'optional_shared_ptr_arra
y<short unsigned int, 4> {aka class object_array_finder<shared_ptr_finder<short
unsigned int, false>, 4>}' has no member named 'bytes'
  int spriteram_bankmax = m_spriteram.bytes() / 0x4000;